sdk/core/azure-core/vcpkg.json

```diff
@@ -1,69 +1,73 @@
 {
-    "name": "azure-core-cpp",
-    "version-string": "1.4.0-beta.1",
-    "dependencies": [
+  "name": "azure-core-cpp",
+  "version-string": "1.4.0-beta.1",
+  "dependencies": [
+    {
+      "name": "openssl",
+      "platform": "!windows & !uwp"
+    },
```

Comment:
There is also a file sdk/core/azure-core/***vcpkg***/vcpkg.json. You need to update it as well, or the vcpkg is going to fail.

This file is for the manifest mode, it is development-time.

The one in the `vcpkg` directory is used when creating a vcpkg and exporting a package.